asf_heat_pump_suitability/pipeline/run_scripts/run_add_features.py

```diff
@@ -151,6 +151,13 @@ def parse_arguments() -> argparse.Namespace:
         lad_cons_areas_df, how="left", left_on="lad_code", right_on="LAD23CD"
     )
 
+    epc_df = epc_df.with_columns(
+        pl.when(pl.col("lad_conservation_area_data_available_ew") == True)
```

Comment:
```suggestion
        pl.when((pl.col("lad_conservation_area_data_available_ew")) & pl.col("COUNTRY").is_in(["England", "Wales"]))
```
I just pushed a change to filter the LAD conservation area data availability results to England and Wales LADs only. Therefore, I'm not sure if we actually need what I've added above, but I think it does add some robustness to the pipeline.